gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Thu Jan 21 14:34:32 IRST 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-rc-2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
```

Comment:
It's updated by Android Studio because incompatible with version 2022.1.1

Reply:
> `gradle-7.6` seems also to be ready based on [Gradle Releases](https://gradle.org/releases/)

It's updated by Android Studio because incompatible with version 2022.1.1